app/screens/emoji_picker/index.tsx

```diff
@@ -1,51 +1,45 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import React, {useCallback, useEffect} from 'react';
-import {Keyboard} from 'react-native';
+import React, {useCallback} from 'react';
+import {DeviceEventEmitter} from 'react-native';
 
-import EmojiPicker from '@components/emoji_picker';
-import useNavButtonPressed from '@hooks/navigation_button_pressed';
-import {dismissModal, setButtons} from '@screens/navigation';
+import {Events} from '@constants';
+import BottomSheet from '@screens/bottom_sheet';
+
+import Picker from './picker';
+import EmojiSectionBar from './picker/footer';
```

Comment:
```suggestion
import PickerFooter from './picker/footer';
```
Match the component name inside `picker/footer`